unix/build_dist.sh

```diff
@@ -64,9 +64,12 @@ done
 cp "$ALS_PROLOG/LICENSE.txt" "$DISTDIR/LICENSE.txt"
 cp -p "$MAN/welcome_standard.txt" "$DISTDIR/README.txt"
 cp -p $MAN/$MANUAL "$DISTDIR/$MANUALNAME"
-mkdir "$DISTDIR/help"
-cp -pr $MAN/$HELP/* "$DISTDIR/help"
-cp -p $MAN/als_help.htm "$DISTDIR/als_help.htm"
+mkdir "$DISTDIR/alshelp"
+cp -pr $MAN/$HELP/* "$DISTDIR/alshelp"
+cp -p $MAN/als_help.html "$DISTDIR/als_help.html"
+cp -p $MAN/alshelp.css "$DISTDIR/alshelp.css"
+cp -p $MAN/package_nav.html "$DISTDIR/package_nav.html"
+
```

Comment:
Ok, took care of all the problems, and pushed html_tokens-fixup branch to github.

On Dec 9, 2015, at 6:02 AM, Chuck Houpt <notifications@github.com> wrote: